guide/source/program_administrator/identify_data_needs.rst

```diff
@@ -301,16 +295,17 @@ immediately.  Follow the steps below to submit additional requests if needed:
    `HPXML GitHub issues page <https://github.com/hpxmlwg/hpxml/issues/>`_.
 #. Click "New Issue"
 #. Fill out the form to ask your question or make your request. No need to assign a person or milestone.
-#. Check the appropriate labels: probably "docs" and "enhancement" or "bug".
+#. Check the appropriate labels: probably "docs" and "enhancement" or "bug". [THESE THREE CATEGORIES SHOULD PROBABLY BE DEFINED SOMEWHERE. THEY ARE NOT EXPLICITLY OBVIOUS AS TO WHEN TO USE WHICH.
 #. Click "Submit New Issue".
+WHAT IF SOMEONE DOESN'T WANT TO CREATE A NEW ACCOUNT OR DOESN'T FEEL COMFORTABLE USING THE SYSTEM (AFTERALL, IT ISN'T THE EASIEST SYSTEM)? CAN THEY EMAIL THEIR ISSUE/SUGGESTION TO SOMEONE? OR, SHOULD THIS TYPE OF THING BE DISALLOWED?
```

Comment:
@ejacobsohn, relating to your question about using GitHub Issues to request features or report bugs, I believe this should be the *only* way to do it. 

In the past it wasn't clear the channel to make these requests and it was a nightmare with people feeling like other groups got preferential treatment for their requests. The process of discussing schema modifications wasn't as transparent as it should have been either. By using GitHub issues as the one place where these discussions happen, it has already solved most of those problems. Our developer community loves it. 

Also, it's not as complicated as what you've been doing to edit this guide. Basically users will write into the GitHub issue what they would have sent in an email to me or Julie. The difference is that the conversation then opens up to the whole working group. Any responses are then emailed back to the original poster and they can reply via email to continue the conversation. It also stays on the list until we do something about it whether that be implement the feature or decide not to take action. Fewer things fall through the cracks.